includes/class-wc-stripe-helper.php

```diff
@@ -583,4 +583,22 @@ public static function should_enqueue_in_current_tab_section( $tab, $section ) {
 
 		return true;
 	}
+
+	public static function should_load_scripts_on_product_page() {
+		$prb_locations = self::get_settings( null, 'payment_request_button_locations' ) ?? [ 'product', 'cart' ];
```

Comment:
Automated checks will continue to fail until we merge #2109 since I'm using the null-coalescing operator `??` here, which is exclusive to PHP 7.0.